django_cryptography/utils/crypto.py

```diff
@@ -18,30 +18,28 @@ class InvalidToken(Exception):
     pass
 
 
-def salted_hmac(key_salt, value, secret=None):
```

Comment:
Then shouldn't we define a CRYPTOGRAPHY_SIGNING_KEY in order to keep parity? This way, we would be able to maintain or change the default behaviour without ignoring what is defined in CRYPTOGRAPHY_KEY. It is my humble opinion, what do you think?